docs/reference/licenses.md

```diff
@@ -14,7 +14,7 @@ Licenses and license information for Zeebe can be found in the Zeebe project [RE
 
 ### Operate, Tasklist, Identity, Optimize
 
-Operate, Tasklist, Identity, and Optimize are licensed to our users and customers under a proprietary license. The proprietary license can be either the [Camunda Platform Self-Managed Free Edition license](https://camunda.com/legal/terms/cloud-terms-and-conditions/camunda-cloud-self-managed-free-edition-terms/) or the Camunda Platform Self-Managed Enterprise Edition license (a copy you obtain when you contact Camunda). Note that while the Camunda Platform Self-Managed Free Edition is indeed free, it only allows usage of the software in non-production environments. To use the software in production, [purchase the Camunda Platform Self-Managed Enterprise Edition](https://camunda.com/products/cloud/camunda-cloud-enterprise-contact/).
+Operate, Tasklist, Identity, and Optimize are licensed to our users and customers under a proprietary license. The proprietary license can be either the [Camunda Platform Self-Managed Free Edition license](https://camunda.com/legal/terms/cloud-terms-and-conditions/camunda-cloud-self-managed-free-edition-terms/) or the Camunda Platform Self-Managed Enterprise Edition license (a copy you obtain when you contact Camunda). Note that while the Camunda Platform Self-Managed Free Edition is indeed free, it only allows usage of the software in non-production environments. To use the software in production, [purchase the Camunda Platform Self-Managed Enterprise Edition](https://camunda.com/products/cloud/camunda-cloud-enterprise-contact/). This licensing information also applies to web apps, APIs, and clients.
```

Comment:
:x: Not sure whether this applies to the clients. Lets wait for @felix-mueller to clarify.

Reply:
Which clients are you referring to? Operate, Tasklist, Optimize do not have clients. Identity has an internal client that is used by Operate, Tasklist, Optimize right?

Reply:
@felix-mueller Yes -- so technically, we can remove clients here. Adjusted in my latest commit.